rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -263,6 +263,13 @@ def _create_networks(self, network_create_args):
     def _bind_ports(self, lports, sandboxes, port_bind_args):
         port_bind_args = port_bind_args or {}
         wait_up = port_bind_args.get("wait_up", False)
+        # "wait_sync" takes effect only if wait_up is True.
+        # By default we wait for all HVs catching up with the change.
+        wait_sync = port_bind_args.get("wait_sync", "hv")
+        if wait_sync not in ['hv', 'sb', 'none']:
+            raise exceptions.InvalidConfigException(_(
```

Comment:
@l8huang I think ovs version check should be executed in ``ovsclients_impl.py``.